docs/CAPTCHA_SETUP.md

```diff
@@ -1,31 +1,35 @@
 # Overview
-Captcha can be enabled for this home server. This file explains how to do that.
-The captcha mechanism used is Google's ReCaptcha. This requires API keys from Google.
-
-## Getting keys
-
-Requires a site/secret key pair from:
-
-<https://developers.google.com/recaptcha/>
-
-Must be a reCAPTCHA v2 key using the "I'm not a robot" Checkbox option
-
-## Setting ReCaptcha Keys
-
-The keys are a config option on the home server config. If they are not
-visible, you can generate them via `--generate-config`. Set the following value:
-
+A captcha can be enabled on your homeserver to help prevent bots from registering
+accounts. Synapse currently uses Google's reCAPTCHA service which requires API keys
+from Google.
+
+## Getting API keys
+
+1. Create a new site at <https://www.google.com/recaptcha/admin/create>
+1. Set the label to anything you want
+1. Set the type to reCAPTCHA v2 using the "I'm not a robot" Checkbox option.
+This is the only type of captcha that works with Synapse.
+1. Add your server name to the list of authorized domains.
```

Comment:
is this server name as in `server_name` as in the thing that ends up in MXIDs, or the public hostname for the C-S API?

I've a feeling it's the latter, in which case it might be better to use words like "the public hostname for your server, as set in `public_baseurl` in `homeserver.yaml`".

Reply:
I was just trying to have them put *some* domain in there because Google requires at least one domain to be added to the list. As long as they do step 8 then it doesn't matter what domains are in the list.

But yes that is probably more correct because someone may not do step 8 and may not have the `public_baseurl` as a subdomain of the server name domain.